zingolib/src/config.rs

```diff
@@ -122,9 +122,8 @@ pub enum ChainFromStringError {
 /// * `Err(String)` - An error message if the chain name is invalid
 pub fn chain_from_str(chain_name: &str) -> Result<ChainType, ChainFromStringError> {
     match chain_name {
-        "testnet" => Ok(ChainType::Testnet),
-        "mainnet" => Ok(ChainType::Mainnet),
-        "regtest" => Err(ChainFromStringError::UnknownRegtestChain),
+        "testnet" | "test" | "regtest" => Ok(ChainType::Testnet),  // Treat regtest/test as testnet
```

Comment:
thanks for you help here, i am going to discuss with the other devs what has happened to the regtest config here. i suspect it was removed due to now using zebra types which include regtest in the testnet chaintype. I will look into this and work out the best way forward as I dont think it was our intention to stop supporting regtest mode in zingo-cli.